ManageIQ/manageiq
# Add load balancer service type

Purpose or Intent
-----------------
Adding LoadBalancer service type and automate state machines. Allowing us to deploy and retire Amazon Load Balancer.

Steps for Testing/QA
--------------------
With Amazon code merged, user should be able to add LoadBalancer service type and deploy it
